File: java/UberBlack.java
import java.util.ArrayList;
import java.util.Map;

class UberBlack extends Car {
    Map<String, Map<String,Integer>> typeCarAccepted;
    ArrayList<String> seatsMaterial;
    
    public UberBlack(String license, Account driver, String brand, String model, Map<String, Map<String,Integer>> typeCarAccepted, ArrayList<String> seatsMaterial) {
        super(license, driver); //user super sugiere parametros principales y llama al metodo constructor de la super clase
        this.seatsMaterial = seatsMaterial;
        this.typeCarAccepted = typeCarAccepted;
    }
}

File: python/uberblack.py
from array import array
from car import Car
class UberBlack(Car):
    typecaraccepted = []
    seatsMaterial = []

    def __init__(self, license, driver,typecaraccepted, seatsMaterial):
        super().__init__(license, driver)
        self.typecaraccepted = typecaraccepted
        self.seatsMaterial = seatsMaterial